src/taco-cli/cli/utils/cordovaHelper.ts

```diff
@@ -159,6 +159,13 @@ class CordovaHelper {
         return CordovaHelper.toCordovaArgumentsInternal(commandData, platforms);
     }
 
+    public static toCordovaTargetsArguments(commandData: commands.ICommandData, platforms: string[] = null): Cordova.ICordovaRawOptions {
+        // Run, build, emulate, prepare and compile all use the same format at the moment
+        return CordovaHelper.toCordovaArgumentsInternal(commandData, platforms);
```

Comment:
Different commands may care about different sets of arguments, but at the moment they do go through a common place in the cordova code so we may be overengineering.